platforms/software/dependency-management/src/main/java/org/gradle/internal/locking/MissingLockStateException.java

```diff
@@ -27,11 +28,11 @@ public class MissingLockStateException extends RuntimeException implements Resol
     private static final List<String> RESOLUTIONS;
     static {
         RESOLUTIONS = ImmutableList.of(
-            "To create the lock state, run a task that will resolve that configuration and add '--write-locks' to the command line.",
+            "To create the lock state, run a task that performs dependency resolution and add '--write-locks' to the command line.",
             new DocumentationRegistry().getDocumentationRecommendationFor("information on generating lock state", "dependency_locking", "generating_and_updating_dependency_locks"));
     }
-    public MissingLockStateException(String configurationName) {
-        super("Locking strict mode: Configuration '" + configurationName + "' is locked but does not have lock state.");
+    public MissingLockStateException(DisplayName displayName) {
+        super("Locking strict mode: " + displayName.getCapitalizedDisplayName() + " is locked but does not have lock state.");
```

Comment:
❔ Wouldn't we also want to see the exact lock id that failed to load here?

Reply:
Perhaps. This does not change the original message, as displayName.getCapitalizedDisplayName() is equal to `Configuration '<configurationName>'`